digits/templates/partials/home/upload_pretrained_model.html

```diff
@@ -1,7 +1,7 @@
 {# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved. #}
 <script type="text/javascript" src="{{ url_for('static', filename='js/components/PretrainedModel.js') }}"></script>
```

Comment:
Don't you want `ver=dir_hash` here? From https://github.com/NVIDIA/DIGITS/pull/904?